test/index.test.ts

```diff
@@ -9,7 +9,32 @@ describe('identity', () => {
 });
 
 describe('make', () => {
-  it('returns `identity`', () => {
+  type PositiveNumber = Brand<number, 'positive'>;
+
+  it('returns `identity` when no validator is provided', () => {
     expect(make<AnyBrand>()).toBe(identity);
   });
+
+  it('utilizes the validator when one is provided with explicit validator types and an implicit brand types', () => {
```

Comment:
```suggestion
  it('uses the validator when one is provided with explicit validator types and an implicit brand type', () => {
```
- "uses" is just a stylistic preference that prefers shorter words.
- one typo at the end